clean-ts/clean-ts.c

```diff
@@ -172,7 +176,6 @@ static int clean_ts(const char *infile, const char *outfile, int64_t npackets,
   if (oc != NULL && oc->pb != NULL) {
     avio_close(oc->pb);
   }
-  avformat_free_context(oc);
```

Comment:
Why did you delete this? `oc` is allocated by `avformat_alloc_output_context2` and it needs to be freed.